### PR TITLE
Add factory reset chapter

### PR DESCRIPTION
**Pull request by gsanchietti**
Card: https://trello.com/c/b1ObeS6M/153-change-filesystem-to-squashfs

### CODE REVIEW

File: reset_recovery.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+However, it's important to note that failsafe mode cannot resolve more complex issues such as faulty hardware or a damaged kernel.
+While it resembles a reset, failsafe mode enables you to access your device and restore settings if necessary, whereas a reset would simply erase all configurations.
+
+The simplest way to activate failsafe mode is to connect directly to the firewall using a VGA/HDMI monitor or a serial cable. To do this,
+boot the machine, wait for the Grub boot menu to appear, and select `NethSecurity (failsafe)`.
+
+You can access the firewall directly through the serial port using a null-modem cable and a common terminal program.
+For Windows, you can use PuTTY version 0.60 or higher. For Linux, options include minicom and picocom. Set the baud rate
+to 115200, data bits to 8, parity to None, and stop bits to 1 (abbreviated as 8N1).
+You can access the firewall directly through the serial port using a null-modem cable and a common terminal program (including PuTTY, version >= 0.60).
```
Comment (filippocarletti) [inline reset_recovery.rst:51]: Repetition: delete 2 two lines.

File: reset_recovery.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+Set the baud rate to 115200, data bits to 8, parity to None, and stop bits to 1 (abbreviated as 8N1).
+
+After entering failsafe mode, the firewall will initiate with a network address of 192.168.1.1/24, usually on the eth0 network interface,
```
Comment (filippocarletti) [inline reset_recovery.rst:54]:
```suggestion
After entering failsafe mode, the firewall will start with a network address of 192.168.1.1/24, usually on the eth0 network interface,
```

File: reset_recovery.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+and only essential services will be operational. It's important to note that the DHCP server will be inactive in failsafe mode.
+Follow the instructions displayed on the screen to mount the root filesystem and access other utilities as needed.
+
+
+.. _recovery-section:
+
+Emergency recovery
+==================
+
+Emergency recovery in NethSecurity, also known as un-bricking, is a  feature enabling users to restore their firewall device in cases of severe malfunctions.
```
Comment (filippocarletti) [inline reset_recovery.rst:64]:
```suggestion
Emergency recovery in NethSecurity, also known as unbricking, is a feature enabling users to restore their firewall device in cases of severe malfunctions.
```

File: reset_recovery.rst [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+Unbricking ensures that even the most critical issues can be resolved, restoring the device to full functionality, unless there are hardware failures.
+
+If you still have access to the system, you can use the following commands to download and write the image: ::
+
+  ns-download-l
```
Comment (filippocarletti) [inline reset_recovery.rst:69]:
```suggestion
  ns-download -l
```

**Review by filippocarletti — CHANGES_REQUESTED** [review]

File: locale/it/LC_MESSAGES/nethsecurity-docs.po [hunk 5]
```diff
@@ -2872,6 +2871,46 @@ msgid ""
...
+#: ../../threat_shield.rst:16
+msgid ""
+"This tool can only be activated for firewalls provided with a "
+"subscription, currently there is no web configuration interface but it is"
+" still possible to configure it from a character interface."
+msgstr ""
+
+#: ../../threat_shield.rst:18
+msgid ""
+"How to configure this service  is explained in the `developer manual "
```
[the remark this answers is not in the document]
Comment (gsanchietti) [reply locale/it/LC_MESSAGES/nethsecurity-docs.po:2910]: Do not modify the po files, there are automated tools for this.

File: locale/it/LC_MESSAGES/nethsecurity-docs.po [hunk 6]
```diff
@@ -2872,6 +2871,46 @@ msgid ""
...
+"<https://www.cyberthreatalliance.org>`_. Yoroi blacklists ensure great "
+"effectiveness and high confidence, minimizing the possibility of false "
+"positives."
+msgstr ""
+
+#: ../../threat_shield.rst:16
+msgid ""
+"This tool can only be activated for firewalls provided with a "
+"subscription, currently there is no web configuration interface but it is"
+" still possible to configure it from a character interface."
```
[the remark this answers is not in the document]
Comment (gsanchietti) [reply locale/it/LC_MESSAGES/nethsecurity-docs.po:2905]: Do not modify the po files, there are automated tools for this: you need to modify the source

File: reset_recovery.rst (hunk 1 above)
Comment (gsanchietti) [reply reset_recovery.rst]: Done

**Review by filippocarletti — APPROVED** [review]

**Pull request merged**